var/spack/repos/builtin/packages/trinity/package.py

```diff
@@ -49,9 +49,13 @@ class Trinity(MakefilePackage):
     depends_on("jellyfish")
     depends_on("salmon")
     depends_on("perl+threads")
+    depends_on("autoconf")
+    depends_on("automake")
+    depends_on("libtool")
```

Comment:
These 3 are probably `type='build'`. `perl` and `java` are probably `type=('build', 'run')`, although you'll have to check.